policy/modules/system/systemd.te

```diff
@@ -671,6 +674,7 @@ sysnet_manage_config(systemd_hostnamed_t)
 systemd_log_parse_environment(systemd_hostnamed_t)
 
 # Allow reading /run/udev/data/+dmi:id
+udev_search_runtime(systemd_hostnamed_t)
 udev_read_runtime_files(systemd_hostnamed_t)
```

Comment:
I guess I missed this before, but this indicates that `udev_read_runtime_files()` is missing perms, since it should already provide `udev_search_runtime()`.

Reply:
That appears to be the case. I will move this access into that interface.